nix/nixos/cardano-db-sync-service.nix

```diff
@@ -217,13 +217,23 @@ in {
               LEDGER_OPTS="--state-dir ${cfg.stateDir}"
             fi
 
-            exec ${exec} \
-                --config ${configFile} \
-                --socket-path "$CARDANO_NODE_SOCKET_PATH" \
-                --schema-dir ${self.schema} \
-                ${toString cfg.rtsArgs} \
-                ''${LEDGER_OPTS} \
-                ''${EXTRA_DB_SYNC_ARGS:-}
+            if [[ -z "''${CARDANO_DB_SYNC_CONFIG:-}" ]]; then
```

Comment:
Let's rename this to `DB_SYNC_CONFIG` to be consistent with the other environment variables